scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtConfig.scala

```diff
@@ -232,6 +252,12 @@ object ScalafmtConfig {
   implicit lazy val codecEncoder: ConfEncoder[Codec] =
     ConfEncoder.StringEncoder.contramap(_.name)
 
+  implicit val confObjReader: ConfDecoder[Conf.Obj] =
```

Comment:
@olafurpg just curious why `Conf.Obj` decoder isn't available as part of `metaconfig`.